src/components/chip-set/chip-set.tsx

```diff
@@ -698,11 +690,16 @@ export class ChipSet {
                 class="mdc-text-field__icon clear-all-button"
                 tabindex="0"
                 role="button"
-                title={this.clearAllChipsLabel}
+                title={this.clearAllChipsLabel()}
                 aria-label={this.clearAllChipsLabel}
             />
         );
     }
+
+    private clearAllChipsLabel = (): string => {
+        return translate.get('chip-set.clear-all', this.language);
+    };
```

Comment:
This change is very definitely not a `feat`. It's a `refactor`. There's no reason we should trigger the release of a new minor version because of this change, and the change should not be documented as a "new feature" in the changelog.

Reply:
That's right. This was a 🧠💨